### DISCUSSION TITLE
Triva: Scott's Style

### DISCUSSION THREAD

**Discussion by natalie-o-perret** [akhansari/AspFeat#1] [General]
There is an opportunity to use AP here: https://github.com/akhansari/AspFeat/commit/6440045c9f72e4aea885e4e108f5c31d33f5bbbf#diff-f75ec88facf3f9606c453d314af240bd9c099ff53b916819f254fd32c9e261cdR25-R35

**Comment by akhansari** [comment]
Good point, thanks. It's a good time to isolate stuff in a module like `Reflect`.
The next step is to have json body injection with bad request handling.

Functions would be (draft) :
- `httpf:  'RouteValues -> HttpHandler`
- `httpj:  'JsonBody -> HttpHandler`
- `httpfj: 'RouteValues -> 'JsonBody -> HttpHandler`
